src/test/java/org/abstractj/kalium/crypto/StreamTest.java

```diff
@@ -0,0 +1,33 @@
+package org.abstractj.kalium.crypto;
+
+import org.abstractj.kalium.NaCl;
+import org.junit.BeforeClass;
+import org.junit.Test;
+

```

Comment:
I think we should make use of testing vectors for this. See: https://github.com/abstractj/kalium/blob/master/src/test/java/org/abstractj/kalium/fixture/TestVectors.java